tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -356,13 +356,16 @@ def parameters(cls):
         parameters.update({
             'output_blob': StringField(optional=True, default=None, description="Output blob name."),
             'scale': NumberField(optional=True, default=1.0, description="Scale factor for bboxes."),
+            'scales': ListField(optional=True, default=None,
+                                description="List with width and height scale factors for bboxes."),
```

Comment:
@eaidova Here are changes I've made to support several scales for bboxes. I wasn't find a way to pass to `scale` parameter both the number and list of 2 numbers, so I added the new  `scales` parameter. Is it appropriate or I missed some way to do it? Feel free to criticize me

Reply:
there is way how it do it.

1. change type of NumberField to BaseField
2. use [get_or_parse_value](https://github.com/openvinotoolkit/open_model_zoo/blob/75ec8ad4b6c4fd7fb161ae9d2c3056281b2443de/tools/accuracy_checker/accuracy_checker/utils.py#L317) function from utils for value parsing. 

As example, you can use normalization preprocessing, which support both per channel mean/std and single value for all channels
https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/accuracy_checker/preprocessor/normalization.py#L55

Reply:
Your solution is quite elegant and great. Thank you for the idea and detailed instructions!